### PR TITLE
Improve serde related tasks

### PR DESCRIPTION
**Pull request by Licenser**

### CODE REVIEW

File: src/serde/value/owned/de.rs [hunk 1]
```diff
@@ -316,6 +316,7 @@ impl<'de> de::Deserializer<'de> for BorrowedCowStrDeserializer<'de> {
     where
         V: de::Visitor<'de>,
     {
+        dbg!();
```
Comment (sunnygleason) [inline src/serde/value/owned/de.rs]: intentional? 😕

File: src/serde/value/owned/de.rs [hunk 2]
```diff
@@ -275,7 +275,7 @@ impl<'de> serde::Deserializer<'de> for MapKeyDeserializer<'de> {
     {
         self.key
             .into_deserializer()
-            .deserialize_enum(name, variants, visitor)
+            .deserialize_enum(dbg!(name), dbg!(variants), visitor)
```
Comment (sunnygleason) [inline src/serde/value/owned/de.rs]: intentional? 😕

File: .github/workflows/validate.yaml [hunk 3]
```diff
@@ -62,6 +63,7 @@ jobs:
       - name: run tarpaulin
         env:
           RUSTFLAGS: '-C target-cpu=native'
+          PROPTEST_CASES: 512
```
Comment (sunnygleason) [inline .github/workflows/validate.yaml]: :+1:

File: src/serde/value/owned/de.rs (hunk 2 above)
Comment (Licenser) [reply src/serde/value/owned/de.rs]: nope just me being stupid :D good catch thanks!

File: src/serde/value/owned/de.rs (hunk 1 above)
Comment (Licenser) [reply src/serde/value/owned/de.rs]: same as below 😂!

File: .github/workflows/validate.yaml (hunk 3 above)
Comment (Licenser) [reply .github/workflows/validate.yaml]: moatr porp's are always better ;)

**Comment by Licenser** [comment]
note: not done yet still testing to be done on the serde parts

**Comment by Licenser** [comment]
Okay we're now well over 80% and I can't see serde any more :joy: so I'm calling this a day.

**Comment by Licenser** [comment]
added https://github.com/simd-lite/simdjson-rs/issues/68 as a followup

File: src/serde/value/owned/se.rs [hunk 4]
```diff
@@ -657,4 +657,124 @@ impl serde::ser::SerializeStructVariant for SerializeStructVariant {
...
+mod test {
+    use crate::serde::from_slice;
+    /*
+    use crate::{
+        owned::to_value, owned::Object, owned::Value, to_borrowed_value, to_owned_value,
+        Deserializer,
+    };
+    use halfbrown::HashMap;
+    use proptest::prelude::*;
+    */
```
Comment (sunnygleason) [inline src/serde/value/owned/se.rs]: curious, why are we keeping this code around? not a blocker!

File: src/serde/value/owned/se.rs [hunk 5]
```diff
@@ -657,4 +657,124 @@ impl serde::ser::SerializeStructVariant for SerializeStructVariant {
...
+    enum Enum {
+        Opt1,
+        Opt2,
+    }
+    impl std::default::Default for Enum {
+        fn default() -> Self {
+            Self::Opt1
+        }
+    }
+    */
```
Comment (sunnygleason) [inline src/serde/value/owned/se.rs]: ditto - curious, why are we keeping this code around? not a blocker!

**Pull request merged**